tests/p2p/test_split_brain.py

```diff
@@ -140,17 +137,15 @@ def test_split_brain_only_blocks_different_height(self):
         self.assertConsensusValid(manager2)
         self.assertConsensusEqual(manager1, manager2)
 
-        self.assertEqual(block_tip1, manager1.tx_storage.indexes.height.get_tip())
-        self.assertEqual(block_tip1, manager2.tx_storage.indexes.height.get_tip())
+        self.assertEqual(block_tip1, not_none(manager1.tx_storage.indexes).height.get_tip())
+        self.assertEqual(block_tip1, not_none(manager2.tx_storage.indexes).height.get_tip())
 
     # XXX We must decide what to do when different chains have the same score
     # For now we are voiding everyone until the first common block
-    def test_split_brain_only_blocks_same_height(self):
+    def test_split_brain_only_blocks_same_height(self) -> None:
         manager1 = self.create_peer(self.network, unlock_wallet=True)
-        manager1.avg_time_between_blocks = 3
 
         manager2 = self.create_peer(self.network, unlock_wallet=True)
-        manager2.avg_time_between_blocks = 3
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
This line was commented instead. So, we are going to fix it later.